src/config/firebase.ts

```diff
@@ -1,9 +0,0 @@
-import admin from 'firebase-admin';
-
-import * as firebaseAccountCredentials from './service.json';
-
-const serviceAccount = firebaseAccountCredentials as admin.ServiceAccount;
-
-export default {
-  credential: admin.credential.cert(serviceAccount),
-};
```

Comment:
Why is this changed?

Reply:
service.json doesnot exist and this file was not imported anywhere else

Reply:
so this was giving error

Reply:
@Abhishek-kumar09 it seems like i had forgot to set up the key.. now i've restored this by adding the service.json